packages/react-flow-editor/src/Editor/components/Output/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { isEqual } from "lodash"
+import React, { useContext } from "react"
+import { Node, NodeState, Output as OutputType } from "@/types"
+import { BUTTON_LEFT } from "../../constants"
+import { EditorContext, RectsContext } from "../../context"
+import { resetEvent } from "../../helpers"
+import { DragItemType } from "../../types"
+import { DragItemAtom, NewConnectionAtom, nodeActions, SvgOffsetAtom, TransformationMap } from "@/Editor/state"
+import { useStore } from "@nanostores/react"
+import { getRectFromRef } from "@/Editor/helpers/getRectFromRef"
```

Comment:
I think we have problems with import's order

Reply:
yep, im add eslint import/order rule